library/src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -223,17 +223,8 @@ private void fireRequestBlockOne() throws RemoteServerException, LoginFailedExce
 	 * @throws RemoteServerException When server fails
 	 */
 	public void fireRequestBlockTwo() throws RemoteServerException, LoginFailedException {
-		ServerRequest[] requests = CommonRequest.fillRequest(
-				new ServerRequest(RequestTypeOuterClass.RequestType.GET_ASSET_DIGEST,
-				CommonRequest.getGetAssetDigestMessageRequest()), this);
-
-		getRequestHandler().sendServerRequests(requests);
-		try {
-			inventories.updateInventories(GetInventoryResponse.parseFrom(requests[2].getData()));
-			settings.updateSettings(DownloadSettingsResponse.parseFrom(requests[4].getData()));
-		} catch (InvalidProtocolBufferException e) {
-			throw new RemoteServerException();
-		}
+		fireRequestBlock(new ServerRequest(RequestTypeOuterClass.RequestType.GET_ASSET_DIGEST,
+				CommonRequest.getGetAssetDigestMessageRequest()));
```

Comment:
we will need to add the code from @ramarro123 to parse the response and update the classes